packages/protocol/contracts/L1/verifiers/IVerifier.sol

```diff
@@ -15,6 +15,8 @@ interface IVerifier {
         uint64 blockId,
         address prover,
         bool isContesting,
+        bool usingBlob,
+        bytes32 blobHash,
```

Comment:
Maybe cleaner to make it part of the `BlockEvidence`?

Reply:
Keeping it in the `BlockEvidence`so that API stay same (+ data related to evidence), i think too might be a good idea.

Reply:
Both  `usingBlob` and `blobHash` are sloaded from storage when a block is to be proven. If we move blobHash into evidence, we still need to check `evicence.blobHash == blobHash`, I don't think it's a good idea to request the prover to provide duplicate datea in evidence.

Reply:
What if put `blobHash` into the `metaHash` , saving some gas ? So that we do not need to store the `blk.blobHash` but it is done indirectly via `metaHash`  (`function hashMetadata(TaikoData.BlockMetadata memory meta`)) ?

Reply:
> I don't think it's a good idea to request the prover to provide duplicate datea in evidence.

The prover doesn't necessarily has to provide the same `TaikoData.BlockEvidence` struct as struct used in `IVerifier`, but yeah that does mean creating a new struct and copying some values over to the `IVerifier` one.